bin/install_pfs.sh

```diff
@@ -3,93 +3,66 @@
 #
 # Install the PFS 2D pipeline code.
 #
-# The PFS 2D pipeline is based on the LSST software stack, which is itself
-# based on Anaconda so there's a lot to install. When it's done you should
-# source <PREFIX>/pfs_setups.sh to set up your environment.
+# The PFS 2D pipeline is based on the LSST software stack, which we
+# install first.
 #
 
-set -ev
-
-# Configuration
-CONDA_VERSION=4.2.12  # Version of Conda to install
-GIT_LFS_VERSION=1.4.4  # Version of git-lfs to install
-LSST_SOURCE=http://conda.lsst.codes/stack/0.12.1  # Source of LSST stack
-LSST_VERSION=lsst-v12_1  # Version of LSST stack to install
-CONDA_FLAVOR=  # Flavor (e.g., Linux vs Mac) of Conda to install
-CONDA_MD5=  # Expected MD5 of Conda installation tarball
-GIT_LFS_FLAVOR=  # Flavor (e.g., Linux vs Mac) of git-lfs to install
-case $(uname -s) in
-    Darwin)
-        CONDA_FLAVOR=MacOSX-x86_64
-        CONDA_MD5=ff3d7b69e32e1e4246176fb90f8480c8
-        GIT_LFS_FLAVOR=darwin-amd64
-        ;;
-    Linux)
-        CONDA_FLAVOR=Linux-x86_64
-        CONDA_MD5=c8b836baaa4ff89192947e4b1a70b07e
-        GIT_LFS_FLAVOR=linux-amd64
-        ;;
-    *)
-        echo "Unrecognised system name: $(uname -s)"
-        exit 1
-esac
```

Comment:
If I understand this correctly, this removes the anaconda installation procedure, assuming that the user has to have it by the time lsst stack is installed? No need to reply if this is correct.

Reply:
Yes, the explicit Conda-based installation is gone. The LSST newinstall.sh (which we're changing over to) uses Conda, but only for the third-party packages.